File: tests/php/testcases/AddFloatsTest.php
<?
require_once 'base.php';

class AddFloatsTestCase extends TestServiceTestCase {

    public function test_null() {
        $this->assertEquals($this->service->add_floats(null, null), 0);
    }

    public function test() {
        $this->assertEquals($this->service->add_floats(10.5, 5.3), 15.8);
    }

}

File: tests/mono/testcases/AddFloatsTest.cs
using System;
using NUnit.Framework;

namespace TestCases {

    [TestFixture()]
    public class AddFloatsTest : TestServiceTest {

        [Test()]
        public void test_null() {
            Assert.AreEqual(service.add_floats(null, null), 0);
        }

        [Test()]
        public void test_empty() {
            Assert.AreEqual(service.add_floats(0, 0), 0);
        }

        [Test()]
        public void test() {
            Assert.AreEqual(
                service.add_floats((float)10.5, (float)5.3), (float)15.8);
        }
    }
}
